injectedJavaScript/application.js

```diff
@@ -1,86 +1,70 @@
-      signaturePadCanvas.width = canvasWidth;
-      signaturePadCanvas.height = canvasHeight;
-      signaturePadCanvas.getContext("2d").scale(devicePixelRatio, devicePixelRatio);
-    };
+  var signaturePad = new SignaturePad(canvas, {
+    penColor: "${penColor || "black"}",
+    backgroundColor: "${backgroundColor || "white"}",
+    onEnd: function() { finishedStroke(signaturePad.toDataURL()); },
+    minWidth: ${penMinWidth || 1},
+    maxWidth: ${penMaxWidth || 4}
+  });
 
-    var enableSignaturePadFunctionality = function () {
-      var signaturePad = new SignaturePad(signaturePadCanvas, {
-        penColor: "${penColor || "black"}",
-        backgroundColor: "${backgroundColor || "white"}",
-        onEnd: function() { finishedStroke(signaturePad.toDataURL()); }
-      });
-      signaturePad.minWidth = ${penMinWidth || 1};
-      signaturePad.maxWidth = ${penMaxWidth || 4};
-      if ("${dataURL}") {
-        signaturePad.fromDataURL("${dataURL}");
-      }
-    };
-
-    sizeSignaturePad();
-    enableSignaturePadFunctionality();
-  };
-
-  var bodyWidth = document.body.clientWidth * 2;
-  var bodyHeight = document.body.clientHeight * 2;
-  if(!bodyWidth) {
-    bodyWidth = window.innerWidth ? window.innerWidth : ${width};
-  }
-  if(!bodyHeight) {
-    bodyHeight = window.innerHeight ? window.innerHeight : ${height};
+  if ("${dataURL}") {
+    signaturePad.fromDataURL("${dataURL}");
   }
+};
 
-  var canvasElement = document.querySelector("canvas");
+function resizeCanvas(canvas) {
+  var devicePixelRatio = Math.max(window.devicePixelRatio || 1, 1);
+  canvas.width = canvas.offsetWidth * devicePixelRatio;
+  canvas.height = canvas.offsetHeight * devicePixelRatio;
+  canvas.getContext("2d").scale(devicePixelRatio, devicePixelRatio);
+}
 
-  var finishedStroke = function(base64DataUrl) {
-    executeNativeFunction('finishedStroke', { base64DataUrl: base64DataUrl });
-  };
+function finishedStroke(base64DataUrl) {
+  executeNativeFunction('finishedStroke', { base64DataUrl: base64DataUrl });
+};
 
-  if (${useFont}) {
-    var context = canvasElement.getContext("2d");
-    var devicePixelRatio = 1; /* window.devicePixelRatio || 1; */
-    canvasElement.width = bodyWidth * devicePixelRatio;
-    canvasElement.height = bodyHeight * devicePixelRatio;
+if (${useFont}) {
+  var context = canvas.getContext("2d");
+  resizeCanvas(canvas);
 
-    var w = bodyWidth;
-    var h = bodyHeight;
-    canvasElement.width = canvasElement.offsetWidth * 2;
-    canvasElement.height = canvasElement.offsetHeight * 2;
+  var w = canvas.offsetWidth;
+  var h = canvas.offsetHeight;
 
-    var fontToHeightRatio = 45 / 159;
-    var fontSize = canvasElement.height * fontToHeightRatio;
-    var textHeight = 18;
-    var textWidth = -1;
-    do {
-      context.font = fontSize + "px SignatureFont, cursive";
-      textWidth = context.measureText("${name}").width;
-      fontSize = 7 * fontSize / 8;
-    } while (textWidth + (w * 0.05) > w);
```

Comment:
This was introduced in this commit: https://github.com/rpmsoftware/react-native-signature-pad/commit/d73c49136b39a6d5ef507a2b2a50f98d61cfe83c

I think the `.05` was a padding around the signature? why remove it?

Reply:
Honestly I was trying to match the behaviour from the RPM web platform to this code. On the web, RPM doesn't have the `0.05` for padding. In testing it myself, the `0.05` wasn't necessary because the text always seem to resize before it even got close to the edges of the canvas.